Update README with merch ordering instructions (#100)
open-life-science/branding: prints-and-digital/README.md

- Social Media Assets

> - letterheads

Reviewer: Please could the capitalisation be made consistent (so letterheads -> Letterheads)? It's small but really bothering me :laughing:

Author: this was already like that- but I'll amend it

Author: done

2. Do not alter the core elements of the assets, such as logos and primary colors, without approval from the Branding Team.
3. When using social media assets, follow the recommended dimensions and formats for each platform to optimize visibility.

## How to order merch:

Different team members have used diffferent providers, and these are some on the ones we have used in the past: 
- https://www.totalmerchandise.co.uk/branded-products/promotional-lifestyle-and-outdoor/seed-sticks-5-pack
- https://zapcreatives.com/
- Hippoprint

> - Moo

Reviewer: Please use hyperlink text as it's more screen reader accessible! So for example, `[Zap Creatives](https://zapcreatives.com/)`. :slightly_smiling_face: Also, are there links for Hippoprint and Moo?

Author: done

> We make an intentional effort to avoid Stickermule since it has been associated with unethical behaviour and practices that we, from OLS do not support. 

Reviewer: Would it be possible to say what exactly kind of practices we consider unethical (lack of sustainability, modern slavery, etc.) and then have a bulleted list of platforms we're avoiding? It's only one platform for now but there will probably be others added to the list eventually!

I'm also a bit worried that the current phrasing could be perceived as libellous. Could it be rephrased to avoidance due to _potentially_ or _possibly_ unethical practices? And an explicit statement that the list is based on community opinion, not necessarily that of OLS or any specific individual associated with OLS?

Author: you're right that this statement is more fitting to the private repo due to its nature, at least for now. I feel like what you're suggesting requires a whole team/community discussion (which I'm up for if needed), but I just wanted to archive this info for now.

Re stickermule, it's knowledge gained form several communities, that I think might get lost as people leave them , that's why I wanted to highlight it here (or at least in internal repo). 

Reason I know of (but didn't research myself) are: 

_They donated $500K to Donald Trump's 2024 campaign, sent mass mailers to their customers in support of Trump, and their CEO is super vocally transphobic.
https://www.theverge.com/2024/7/17/24200441/sticker-mule-trump-endorsement-rally-shooting
https://slate.com/business/2024/07/sticker-mule-ceos-pro-trump-maga-email-surprised-employees.html
https://www.opensecrets.org/outside-spending/detail/2024?cmte=C00887703&tab=targeted_candidates_

Author: Like, I think it feels more like a policy thing, which might be needed- but again, I just wanted to archive historical knowledge for now 😅

Reviewer: Definitely agree it's a discussion to bring to the community! In that case, maybe it could be updated to:

> We make an intentional effort to avoid Sticker Mule since it has _**allegedly**_ been associated with unethical behaviour and practices that OLS does not support.

Adding "allegedly" removes the liability on OLS while still archiving the community's thoughts on the matter.

> We make an intentional effort on not using Stickermule since it has been associated with unethical behaviour and practices that we, from OLS do not support. 

Reviewer 2: ```suggestion
We make an intentional effort to avoid Stickermule since it has been associated with unethical behaviour and practices that we, from OLS do not support. 
```

> Please have in mind this kind of considerations when choosing providers, if ppossibel at all. 

Reviewer 2: ```suggestion
Please have in mind this kind of considerations when choosing providers, if possible at all. 
```
